src/utils.ts

```diff
@@ -49,6 +49,19 @@ export const assembleSecondaryButton = (
   };
 };
 
+export const assembleCategoryBlock = (
+  category: CookieConsentCategoryValues,
+  title: string,
+  description: string,
+  readonly: boolean,
+): VanillaCookieConsent.ModalBlock => {
+  return {
+    title: title,
```

Comment:
```suggestion
    title,
```

You can use a shorthand when the key and variable name are the same name.